pkg/operator/controllers/banner/banner.go

```diff
@@ -0,0 +1,47 @@
+package banner
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+)
+
+const (
+	BannerName         = "openshift-aro-sre"
+	TextContactSupport = "Please ask your cluster administrator to contact Azure or Red Hat support."
+)
+
+func (r *BannerReconciler) reconcileBanner(ctx context.Context, instance *arov1alpha1.Cluster) error {
+	var err error
+	switch instance.Spec.Banner.Content {
+	case arov1alpha1.BannerEmpty:
+		_ = r.consolecli.ConsoleV1().ConsoleNotifications().Delete(ctx, BannerName, metav1.DeleteOptions{})
```

Comment:
err lost?

Reply:
If the Banner content is set to 0 there's a "not found" error on every reconcile, I want to avoid having a Get -> if exists ->Delete because even then you can get a time of check, time of use race. Also every error returned from the function is logged which is in general a good idea. 

This causes a lot of errors in the log that aren't actual errors.

Reply:
You should still handle errors. In this case you probably need logic like:

* Try to delete
  * If delete returns "not found erro" - treat it as not error
  * If delete returns anything else - this is a legitimate error

Here is a similar case:

https://github.com/Azure/ARO-RP/blob/e4ff2a4328ccf77aa600d332364ae8fd4f7b28d9/pkg/operator/controllers/pullsecret/pullsecret_controller.go#L179-L185

Reply:
Ack, added the if statement to nillify the error if it's an "Not Found".